ppafm/cli/relaxed_scan.py

```diff
@@ -46,146 +48,152 @@ def main():
+        ff_kpfm_tvs0 = ff_kpfm_tvs0 * opt_dict["pol_t"]
+
+    lvec[1, :] = rotate_vector(lvec[1, :], opt_dict["rotate"])
+    lvec[2, :] = rotate_vector(lvec[2, :], opt_dict["rotate"])
+    common.lvec2params(lvec)
+
+    for charge, stiffness, voltage in it.product(charges, k_constants, voltages):
+        common.params["charge"] = charge
+        common.params["klat"] = stiffness
+        common.params["Vbias"] = voltage
+
+        dirname = f"Q{charge:1.2f}K{stiffness:1.2f}"
+        if applied_bias:
+            dirname += f"V{voltage:1.2f}"
+        print(" Relaxed_scan for ", dirname)
+        if not os.path.exists(dirname):
+            os.makedirs(dirname)
+
+        # Run relaxation
+        fzs, pp_positions, pp_displacements, lvec_scan = perform_relaxation(
+            lvec,
+            ff_vdw,
+            FFel=ff_electrostatics,
+            FFpauli=ff_pauli,
+            FFboltz=ff_boltzman,
+            FFkpfm_t0sV=ff_kpfm_t0sv,
+            FFkpfm_tVs0=ff_kpfm_tvs0,
+            tipspline=args.tipspline,
+            bFFtotDebug=args.bDebugFFtot,
+        )
+
+        data_info = {"lvec": lvec_scan, "data_format": args.output_format, "head": atomic_info_or_head, "atomic_info": atomic_info_or_head}
+        if common.params["tiltedScan"]:
+            io.save_vec_field(dirname + "/OutF", fzs, **data_info)
+        else:
+            io.save_scal_field(dirname + "/OutFz", fzs, **data_info)
+
+        if opt_dict["vib"] >= 0:
+            which = opt_dict["vib"]
+            print(f" === Computing eigenvectors of dynamical matrix: which={which} ddisp={common.params['ddisp']}")
+            x_tips, y_tips, z_tips, lvec_scan = common.prepareScanGrids()
+            r_tips = np.array(np.meshgrid(x_tips, y_tips, z_tips)).transpose(3, 1, 2, 0).copy()
+            evals, evecs = core.stiffnessMatrix(r_tips.reshape((-1, 3)), pp_positions.reshape((-1, 3)), which=which, ddisp=common.params["ddisp"])
+            io.save_vec_field(dirname + "/eigvalKs", evals.reshape(r_tips.shape), **data_info)
+            if which > 0:
+                io.save_vec_field(dirname + "/eigvecK1", evecs[0].reshape(r_tips.shape), **data_info)
+            if which > 1:
+                io.save_vec_field(dirname + "/eigvecK2", evecs[1].reshape(r_tips.shape), **data_info)
+            if which > 2:
+                io.save_vec_field(dirname + "/eigvecK3", evecs[2].reshape(r_tips.shape), **data_info)
+
+        if opt_dict["disp"]:
+            io.save_vec_field(dirname + "/PPdisp", pp_displacements, **data_info)
+
+        if opt_dict["pos"]:
+            io.save_vec_field(dirname + "/PPpos", pp_positions, **data_info)
+
+        if args.bI:
+            print("Calculating current from tip to the Boltzmann particle:")
+            i_in, lvec, _, atomic_info_or_head = io.load_scal_field("I_boltzmann", data_format=args.output_format)
+            i_out = interpolate_cartesian(i_in, pp_positions, cell=lvec[1:, :], result=None)
```

Comment:
The meaning of `i_in` and `i_out` are unclear here. With the previous capital `I`, I would assume that these refer to some currents. The lower-case `i` could be anything.

Reply:
done in https://github.com/Probe-Particle/ppafm/pull/215/commits/8b7de26f991732f1e2a2b02cc6fdada6cef46ed9